src/lib.rs

```diff
@@ -488,6 +464,9 @@ decl_storage! {
 
         ReplyById get (fn reply_by_id): double_map hasher(blake2_128) (T::BlogId, T::PostId), blake2_128(T::ReplyId) => Reply<T>;
 
+        /// Mapping, representing AccountId -> All presented reactions state mapping by unique post or reply identificators.
+        Reactions get(reactions): double_map hasher(blake2_128) (T::BlogId, T::PostId, Option<T::ReplyId>), blake2_128(T::ParticipantId) => Vec<bool>;
```

Comment:
- Why is the value `Vec<bool>`, do we not have access to the number of reactions at compile time? in which case we can use an array.

- We only have reactions on blog posts, this key type seems wrong.

Reply:
- Because from the inside of blog module this value does not implement Sized Trait for the size to be known at compile time. Actually, in module we have it situated inside of heap allocated container.
I even have tried to use `lazy_static`, but still had some issues with `Encode`, `Decode` parity scale codec traits https://docs.rs/lazy_static/1.4.0/lazy_static/
- Sorry, but spec says something different. Do we need to rethink this?
![OnPaste 20200403-011224](https://user-images.githubusercontent.com/18070359/78306069-2a3f2600-754b-11ea-977c-4963e6ac99d4.png)

Reply:
- Ok, good to know. Can we then this into a constant that is not configurable in the runtime trait? This has the disadvantage that is someone wants to use the module in a different runtime, they will have to fork for a different value, however, I think the type safety is worth it. One could perhaps make it configurable in some other static way, like accepting something from parameter value defined in the building process? not sure, but lets make it a constant for now.

- I forgot, good that you are on top of things! Then the representation is perfect.

Reply:
Done here https://github.com/Joystream/substrate-blog-module/pull/1/commits/b1746053298d36182288746a74a346072ae4367f